package.json

```diff
@@ -22,6 +22,6 @@
   "homepage": "https://github.com/arccoza/metalsmith-with-postcss",
   "dependencies": {
     "minimatch": "^3.0.0",
-    "postcss": "^5.0.19"
+    "postcss": "^6.0.9"
```

Comment:
Minimatch is fine, it's still on 3.0.4, and isn't the important dep. I actually wanted to replace it with [micromatch](https://github.com/micromatch/micromatch) sometime.